stubs/python-jenkins/jenkins/__init__.pyi

```diff
@@ -0,0 +1,216 @@
+    def get_plugins(self, depth: int = 2) -> Mapping[Any, Any]: ...
+    def get_jobs(
+        self, folder_depth: int = 0, folder_depth_per_request: int = 10, view_name: str | None = None
+    ) -> Sequence[Mapping[str, str]]: ...
+    def get_all_jobs(
+        self, folder_depth: int | None = None, folder_depth_per_request: int = 10
+    ) -> Sequence[Mapping[str, str]]: ...
+    def copy_job(self, from_name: str, to_name: str) -> None: ...
+    def rename_job(self, from_name: str, to_name: str) -> None: ...
+    def delete_job(self, name: str) -> None: ...
+    def enable_job(self, name: str) -> None: ...
+    def disable_job(self, name: str) -> None: ...
+    def set_next_build_number(self, name: str, number: int) -> None: ...
+    def job_exists(self, name: str) -> bool: ...
+    def jobs_count(self) -> int: ...
+    def assert_job_exists(self, name: str, exception_message: str = "job[%s] does not exist") -> None: ...
+    def create_folder(self, folder_name: str, ignore_failures: bool = False) -> None: ...
+    def upsert_job(self, name: str, config_xml: str) -> None: ...
+    def check_jenkinsfile_syntax(self, jenkinsfile: str) -> Sequence[str]: ...
+    def create_job(self, name: str, config_xml: str) -> None: ...
+    def get_job_config(self, name: str) -> str: ...
+    def reconfig_job(self, name: str, config_xml: str) -> None: ...
+    def build_job_url(
+        self, name: str, parameters: Mapping[str, Any] | Sequence[tuple[str, Any]] | None = None, token: str | None = None
+    ) -> str: ...
+    def build_job(
+        self, name: str, parameters: Mapping[str, Any] | Sequence[tuple[str, Any]] | None = None, token: str | None = None
+    ) -> int: ...
+    def run_script(self, script, node: str | None = None) -> str: ...
+    def install_plugin(self, name: str, include_dependencies: bool = True) -> bool: ...
+    def stop_build(self, name: str, number: int) -> None: ...
+    def delete_build(self, name: str, number: int) -> None: ...
+    def wipeout_job_workspace(self, name: str) -> None: ...
+    def get_running_builds(self) -> Sequence[Mapping[str, Any]]: ...
+    def get_nodes(self, depth: int = 0) -> Sequence[Mapping[str, Any]]: ...
+    def get_node_info(self, name: str, depth: int = 0) -> Mapping[str, Any]: ...
+    def node_exists(self, name: str) -> bool: ...
+    def assert_node_exists(self, name: str, exception_message: str = "node[%s] does not exist") -> None: ...
+    def delete_node(self, name: str) -> None: ...
+    def disable_node(self, name: str, msg: str = "") -> None: ...
+    def enable_node(self, name: str) -> None: ...
+    def create_node(
+        self,
+        name: str,
+        numExecutors: int = 2,
+        nodeDescription: str | None = None,
+        remoteFS: str = "/var/lib/jenkins",
+        labels: str | None = None,
+        exclusive: bool = False,
+        launcher: str = "hudson.slaves.CommandLauncher",
+        launcher_params: Mapping[str, Any] = {},
```

Comment:
This mapping as to be mutable because of the line `launcher_params['stapler-class'] = launcher`
```suggestion
        launcher_params: MutableMapping[str, Any] = {},
```